wsdl/ver20/ptz/wsdl/ptz.wsdl

```diff
@@ -1166,7 +1166,12 @@ If no stop argument for pan, tilt or zoom is set, the device will stop all ongoi
 			<wsdl:output message="tptz:GetCompatibleConfigurationsResponse"/>
 		</wsdl:operation>
         <wsdl:operation name="MoveAndStartTracking"> 
-            <wsdl:documentation>Documentation TODO...</wsdl:documentation> 
+            <wsdl:documentation>Operation to send an an atomic command to the device: move the camera to a wanted position and then delegate the PTZ control to the tracking algorithm. <br/>
+                    An existing Speed argument overrides DefaultSpeed of the corresponding PTZ configuration during movement to the requested position. 
+                    If spaces are referenced within the Speed argument, they shall be speed spaces supported by the PTZ node. <br/>
+                    If the detection and the tracking are done in the same device, an ObjectID reference can be passed as an argument, in order to specify which object should be tracked. <br/>
+                    The operation shall fail if the requested absolute position is not reachable.
```

Comment:
Should we have shall in wsdl?

Reply:
This paragraph is copied from the specs description. If it is too long or too verbose we can trim it.
Anyway, "shall" is already mentioned in other descriptions in the same wsdl.

Reply:
Agree with Stefan, that there is a need for a capability. Propose to discuss the capability topic at the next Video Enhancement call. Suggest something along
`<xs:attribute name="MoveAndTrack" type="tt:StringList"/> ` 
with the string list containing **PresetToken**, **GeoLocation**, **PTZVector**, and or **ObjectID**.
And then add related requirements to the MoveAndStartTracking section.

By the way I abbreviated the names a bit because I think that MoveAndStartTrackingSupported is far too long.

Reply:
Agree that we need capability, when doing plugfest also we notified the same to videotec. We can discuss this in video enhancement call.